sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/AlertTriggering/BoundaryMeasureType.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.AI.MetricsAdvisor.Models
+{
+    /// <summary>
+    /// Used as part of a <see cref="MetricBoundaryCondition"/>. Specifies which measure should be
+    /// used when checking boundaries. Defaults to <see cref="BoundaryMeasureType.Value"/>.
+    /// </summary>
+    [CodeGenModel("ValueType")]
+    public readonly partial struct BoundaryMeasureType
+    {
+        /// <summary>
+        /// The value of metric is used as it is.
+        /// </summary>
+        public static BoundaryMeasureType Value { get; } = new BoundaryMeasureType(ValueValue);
+
+        /// <summary>
+        /// The mean of the latest metric values in the time series is used.
```

Comment:
Asked the service how many data points are used in the calculation of this mean. I'll wait for their response before merging this PR.

Reply:
Does the user set the time series somewhere? or is this part of the answer you are waiting for

Reply:
The time series are defined by the user-provided data source that is set during data feed creation.

A time series represents a value that varies over time. It could be, for instance, the amount of electrical energy consumed by a factory in a single day, or the total cost a company had in the period of one month. The value being measured (the "metric") is registered periodically (daily, monthly, etc), and monitored by the service.

An example of time series (plotted as a graph over time):
![image](https://user-images.githubusercontent.com/7576220/123875505-a95a1000-d8ee-11eb-9f9f-4a5bfbe6cd1d.png)

These red spots are anomalies detected by the service. The service, however, will not notify the customer every single time an anomaly is detected. Users need to set an Alert Configuration to be notified. Every time an anomaly is detected in a time series, it will go through this configuration to check if it's an anomaly worth being reported. This new type, `BoundaryMeasureType`, is used as part of this Alert Configuration. Every time we mention a "time series" in this context, it should be interpreted as "the time series in which the current anomaly was detected".